php/README.md

```diff
@@ -130,31 +129,39 @@ We recommend that you embed the source code in your container and copy it to the
 |  `fpm.templates` | Additional ConfigMap as a string to be passed to the tpl function. | setting `php-fpm.conf`、`php-fpm.d/www.conf` |
 |  `fpm.annotations` | Grant annotations to ConfigMap of `fpm.templates`, Secrets of `fpm.secrets` | `{}` |
 
-### Default PHP-FPM Configuration
+### Manage Nginx host and port
+When changing `host` and `port` of `nginx.conf`,
+change `host` and `port` of `templates.conf.d.default.conf`.
 
+```
+  templates:
+    conf.d:
+      default.conf: |
+        server {
+            listen      80;
+            server_name localhost;
+```
 
-In the PHP container, you can make some settings by default.
-If you want to set other values​​, please overwrite `fpm.templates`.
-
-For details of each value, please see [here](http://php.net/manual/en/install.fpm.configuration.php).
-
+### Manage Request Timeout Setting
+Change each value to perform graceful shutdown safely according to your requirements.
+#### Nginx
+change the following items.
+`.Values.nginx.lifecycle.preStop`
 
-#### Global Configuration
+default value is `sleep 0`
+```
+  lifecycle:
+    postStart: []
+    preStop: ["/bin/sh", "-c", "sleep 0; nginx -s quit; sleep 5"]
+```
 
-|  Parameter | Default |
-| --- | --- |
-|  `fpm.logLevel` | `notice` |
-|  `fpm.emergencyRestartThreshold`  | `0` |
-|  `fpm.emergencyRestartInterval` | `0` |
-|  `fpm.processControlTimeout` | `0` |
+#### PHP-FPM
+change the following items.
+`.Values.fpm.lifecycle.preStop`
 
-#### Pool Configuration
-|  Parameter | Default |
-| --- | --- |
-|  `fpm.user` | `nobody` |
-|  `fpm.group` | `nobody` |
-|  `fpm.mode` | `600` |
-|  `fpm.pm.maxChildren` | `5` |
-|  `fpm.pm.maxRequests` | `500` |
-|  `fpm.pm.status` | `/status.php` |
-|  `fpm.requestTerminateTimeout` | `0` |
+default value is `sleep 0`
+```
+  lifecycle:
+    postStart: []
+    preStop: ["/bin/sh", "-c", "sleep 1; kill -QUIT 1; sleep 0"]
+```
```

Comment:
@cw-niinuma 
The value of `process_control_timeout` is important for a safe shutdown. Next, please add to README.md.
https://github.com/chatwork/charts/pull/109/files#diff-18bf1c8a8cbc10309f2eca7c8c03fde3R518